src/features/calendar/components/EventCluster/Arbitrary.tsx

```diff
@@ -112,6 +112,7 @@ const Arbitrary: FC<ArbitraryProps> = ({
   return (
     <Event
       cancelled={anyEventIsCancelled}
+      eventList={events}
```

Comment:
A single event should not have to care about a list of events, so the `eventList` property is a separation of concerns red flag for me.